lib/uwb/UwbPeer.cxx

```diff
@@ -1,10 +1,30 @@
 
 #include <tuple>
 
+#include <sstream>
+#include <string>
 #include <uwb/UwbPeer.hxx>
-
 using namespace uwb;
 
+std::string
+UwbPeerSpatialProperties::ToString() const
```

Comment:
```suggestion
UwbPeerSpatialProperties::ToString() const noexcept
```

Reply:
isn't it possible for the string allocation to fail?

Reply:
Yes, good call, ignore.